src/backend/nodes/outfast.c

```diff
@@ -528,7 +528,7 @@ _outQuery(StringInfo str, Query *node)
 	WRITE_NODE_FIELD(rowMarks);
 	WRITE_NODE_FIELD(setOperations);
 	WRITE_NODE_FIELD(constraintDeps);
-	WRITE_BOOL_FIELD(parentStmtType);
+	WRITE_ENUM_FIELD(parentStmtType, ParentStmtType);
```

Comment:
`ParentStmtType` is uint8. I think, it is incorrect macro for this type. You can change `ParentStmtType` or create macro for uint8.

Reply:
fixed